nalgebra-sparse/src/csr.rs

```diff
@@ -7,7 +7,9 @@
 mod csr_serde;
 
 use crate::cs;
-use crate::cs::{CsLane, CsLaneIter, CsLaneIterMut, CsLaneMut, CsMatrix};
+use crate::cs::{
+    CsInnerIter, CsInnerIterMut, CsLane, CsLaneIter, CsLaneIterMut, CsLaneMut, CsMatrix,
```

Comment:
Forgive me if I'm wrong, but I think `CsInnerIter` is exposed in the API but it's not actually publicly exported? I guess the compiler warns about this?